recipes/flatcc/all/test_package/conanfile.py

```diff
@@ -0,0 +1,27 @@
+import os.path
+
+from conans import ConanFile, CMake, tools, RunEnvironment
+
+
+class FlatccTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        env_build = RunEnvironment(self)
+        with tools.environment_append(env_build.vars):
+            cmake = CMake(self)
+            if tools.os_info.is_macos and self.options["flatcc"].shared:
+                # Because of MacOS System Integraty Protection it is currently not possible to run the flatcc
+                # executable from cmake if it is linked shared. As a temporary work-around run flatcc here in
+                # the build function.
+                tools.mkdir(os.path.join(self.build_folder, "generated"))
+                self.run("flatcc -a -o " + os.path.join(self.build_folder, "generated") + " " + os.path.join(self.source_folder, "monster.fbs"))
```

Comment:
```suggestion
                self.run("flatcc -a -o " + os.path.join(self.build_folder, "generated") + " " + os.path.join(self.source_folder, "monster.fbs"), run_environment=True)
```

Reply:
Thanks @madebr, it was probably too late for me...